components/sup/src/command/start.rs

```diff
@@ -85,61 +84,71 @@ static LOGKEY: &'static str = "CS";
-                        // It is important to pass `config.package()` to `show_package()` instead of the
-                        // package identifier of the loaded package. This will ensure that if the operator
-                        // starts a package while specifying a version number, they will only automaticaly
-                        // receive release updates for the started package.
-                        //
-                        // If the operator does not specify a version number they will automatically receive
-                        // updates for any releases, regardless of version number, for the started  package.
-                        let depot_client = try!(Client::new(url, PRODUCT, VERSION, None));
-                        let latest_pkg_data =
-                            try!(depot_client.show_package((*config.package()).clone()));
-                        let latest_ident: PackageIdent = latest_pkg_data.get_ident().clone().into();
-                        if &latest_ident > package.ident() {
-                            outputln!("Downloading latest version from remote: {}", latest_ident);
-                            let mut progress = ProgressBar::default();
-                            let archive = try!(depot_client.fetch_package(latest_ident,
-                                               &cache_artifact_path(None),
-                                               Some(&mut progress)));
-                            try!(archive.verify(&default_cache_key_path(None)));
-                            try!(archive.unpack(None));
-                        } else {
-                            outputln!("Already running latest.");
-                        };
-                    }
+                    let url = config.url();
+                    outputln!("Checking remote for newer versions...");
+                    // It is important to pass `config.package()` to `show_package()` instead
+                    // of the package identifier of the loaded package. This will ensure that
+                    // if the operator starts a package while specifying a version number, they
+                    // will only automaticaly receive release updates for the started package.
+                    //
+                    // If the operator does not specify a version number they will
+                    // automatically receive updates for any releases, regardless of version
+                    // number, for the started  package.
+                    let depot_client = try!(Client::new(url, PRODUCT, VERSION, None));
+                    let latest_pkg_data =
+                        try!(depot_client.show_package((*config.package()).clone()));
+                    let latest_ident: PackageIdent = latest_pkg_data.get_ident().clone().into();
+                    if &latest_ident > package.ident() {
+                        outputln!("Downloading latest version from remote: {}", latest_ident);
```

Comment:
same here re: "remote", although, I don't feel too strongly about it either way.